draft-age-masque-connect-ip.md

```diff
@@ -405,10 +429,38 @@ Payload = Encapsulated IP Packet
 ~~~
 {: #fig-tunnel title="VPN Tunnel Example"}
 
+## IP Flow Proxying
+
```

Comment:
Given that we've agreed that "ICMP, here be dragons" can we perhaps use SCTP in this example instead of ICMP?